### PR TITLE
feature(EJ2-67745): Upgraded packages and integrated Syncfusion components

### PR DESCRIPTION
**Pull request by manikandan-mariappan**
Feature Description: 
To upgrade syncfusion packages and to use Syncfusion components wherever applicable

Solution Description:
Update all packages to latest version and Integrate following Syncfusion Components
- AppBar
- Avatar
- Card
- Skeleton

Output Screenshots:

**Before AppBar, Avatar and Card**

BeforeAppBarCards

**After AppBar, Avatar and Card**

AfterAppBarCards

**Implemention of Skeleton Component**

InitialSkeleton

**Responsive View**

HTMobileView

SkeletonMobile

SkeletonTablet

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,45 +14,47 @@
...
+        "@syncfusion/ej2-angular-charts": "*",
         "@syncfusion/ej2-angular-dropdowns": "*",
+        "@syncfusion/ej2-angular-grids": "*",
+        "@syncfusion/ej2-angular-inputs": "*",
+        "@syncfusion/ej2-angular-layouts": "*",
         "@syncfusion/ej2-angular-navigations": "*",
-        "rxjs": "~6.6.0",
-        "tslib": "^2.0.0",
-        "zone.js": "~0.10.2"
+        "@syncfusion/ej2-angular-notifications": "^20.4.42",
```
Comment (NavinVinayagam) [inline package.json]: Use * for version

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -1,72 +1,71 @@
...
+            <span class="e-avatar custom-avatar e-avatar-circle"><img src="./assets/images/icons/Profile_.png"></span>
+        </div>
+        </ejs-appbar>
     <div id="card-row">
         <div class="row">
             <div class="col-xs-6 col-xl-3 col-lg-3 col-md-3 col-sm-6 calories-valueactive" id="calories-column"
                 style="padding-right: 0px; padding-left: 20px">
-                <div class="card" id="calories" (click)="onCalClick($event)" style="max-width: 445px; background-image: linear-gradient(45deg, #F23B3B 0%, #EF9027 100%)">
-                    <div class="card-body expense-card" id="calories-bg" style=" background-image: linear-gradient(45deg, #F23B3B 0%, #EF9027 100%); border-radius: 4px">
+				<ejs-skeleton cssClass="card-skeleton" width="100%" shimmerEffect="Wave"></ejs-skeleton>
```
Comment (NavinVinayagam) [inline src/app/dashboard/dashboard.component.html]: check alignment

File: src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -276,20 +248,268 @@ export class DashBoardComponent implements OnInit {
...
+    }
+    public LoadCardData(): void {
+        this.GetCardData().then((data: any) => {
+            document.getElementById('calories-text').textContent = data['calories-eaten'];
+            document.getElementById('steps-text').textContent = data['steps-taken'];
+            document.getElementById('water-text').textContent = data['water-consumed'];
+            document.getElementById('sleep-text').textContent = data['sleep-duration'];
+            let cardSkeletons: HTMLCollectionOf<Element> = document.getElementById('card-row').getElementsByTagName('ejs-skeleton');
+            const cards: NodeList = document.querySelectorAll('#card-row .row .e-card');
+            (cardSkeletons[0] as HTMLElement).style.display = 'none';
```
Comment (NavinVinayagam) [inline src/app/dashboard/dashboard.component.ts]: Use Visible property of skeleton instead

File: src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -996,13 +1144,13 @@ export class DashBoardComponent implements OnInit {
         document.getElementById('steps-subtitle').style.color = '#828282';
         document.getElementById('sleep-text').style.color = '#828282';
         document.getElementById('sleep-subtitle').style.color = '#828282';
-        this.polarChart.series[0].dataSource = < Object[]>this.polarchartdata;
+        this.LoadChartData("water");
+        // this.polarChart.series[0].dataSource = < Object[]>this.polarchartdata;
```
Comment (NavinVinayagam) [inline src/app/dashboard/dashboard.component.ts]: remove the comment line

File: src/assets/index.css [hunk 5]
```diff
@@ -377,6 +212,19 @@
 #chart-leftarea {
   margin-left: 3%;
 }
+#calories,#step,#water,#sleep {
+  display: none;
+  border-radius: 4px;
+  cursor: pointer;
+  overflow: visible !important;
```
Comment (NavinVinayagam) [inline src/assets/index.css:278]: Try to avoid important here

File: src/assets/index.css [hunk 6]
```diff
@@ -771,27 +609,46 @@ text-align: center;
...
-      font-family: 'Roboto';
-      font-weight: 400;
-      font-size: 20px !important;
-      color: #727272;
-      margin-bottom: 0vw;
-      margin-top: 10px;
-      letter-spacing: 0;
+  #calories-column, #step-column, #water-column, #sleep-column {
+    width: 100% !important;
+    padding: 0px 15px !important;
```
Comment (NavinVinayagam) [inline src/assets/index.css]: Confirm the need for important

File: src/styles.css [hunk 7]
```diff
@@ -1 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
+@import "../node_modules/@syncfusion/ej2-base/styles/material.css";
```
Comment (NavinVinayagam) [inline src/styles.css]: Use the URL's in same format

**Review by NavinVinayagam — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (manikandan-mariappan) [reply package.json]: Changed to * instead of Particular Version

File: src/styles.css (hunk 7 above)
Comment (manikandan-mariappan) [reply src/styles.css]: Same format has been used

File: src/app/dashboard/dashboard.component.html (hunk 2 above)
Comment (manikandan-mariappan) [reply src/app/dashboard/dashboard.component.html]: Alignments Ensured

File: src/app/dashboard/dashboard.component.ts (hunk 3 above)
Comment (manikandan-mariappan) [reply src/app/dashboard/dashboard.component.ts]: Implemented Conditional Rendering so it is not needed

File: src/app/dashboard/dashboard.component.ts (hunk 4 above)
Comment (manikandan-mariappan) [reply src/app/dashboard/dashboard.component.ts]: Comment lines have been removed

File: src/assets/index.css (hunk 5 above)
Comment (manikandan-mariappan) [reply src/assets/index.css:278]: Avoided unnecessary usage of !important tags

File: src/assets/index.css (hunk 6 above)
Comment (manikandan-mariappan) [reply src/assets/index.css]: Ensured

**Review by NavinVinayagam — APPROVED** [review]

**Review by syncsiva — APPROVED** [review]

**Review by syncsiva — APPROVED** [review]

**Pull request merged**